classes/cluster/mk24-qa-bare-metal-01/openstack/init.yml

```diff
@@ -0,0 +1,110 @@
+parameters:
+  _param:
+    openstack_version: mitaka
+    openstack_region: RegionOne
+    admin_email: root@localhost
+    galera_server_cluster_name: openstack_cluster
+    galera_server_maintenance_password: workshop
+    galera_server_admin_password: workshop
+    rabbitmq_secret_key: workshop
+    rabbitmq_admin_password: workshop
+    rabbitmq_openstack_password: workshop
+    glance_version: ${_param:openstack_version}
+    glance_service_host: ${_param:openstack_control_address}
+    keystone_version: ${_param:openstack_version}
```

Comment:
Wat about   values that need to be defined according to the neutron formula:
  neutron_gateway_agent_mode: 
   neutron_compute_dvr: True
neutron_compute_agent_mode: dvr